src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -566,9 +566,14 @@ export function makeEnrichmentDataPromise<T extends {cytoband?:string, hugoGeneS
                 }
 
                 const sortedByPvalue = _.sortBy(data, c=>c.pValue);
-                const qValues = calculateQValues(sortedByPvalue.map(c=>c.pValue));
-                qValues.forEach((qValue, index)=>{
-                    sortedByPvalue[index].qValue = qValue;
+                const qValues = calculateQValues(sortedByPvalue.filter(c => c.pValue !== undefined).map(c => c.pValue));
+
+                let index = 0;
+                sortedByPvalue.forEach(datum => {
```

Comment:
this might be a little more straightforward if you just saved the result of the filter above and then did map on that, assigning qValue.  not that important.

Reply:
I actually think this is better because that way you dont duplicate the logic of pValue !== undefined. it's not a big deal in this case because it's easy to recompute, but I would say in general that makes the code slightly harder to read and more brittle. I think it's definitely worth doing

Reply:
I could, but the results contain everything (even when pValue is undefined)